modules/serverless-pingsource-odc.adoc

```diff
@@ -39,9 +39,17 @@ spec:
 .. Click *Create*.
 
 . Create a PingSource in the same namespace as the service created in the previous step, or any other sink that you want to send events to.
-.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*.
-.. Select *Ping Source*.
-.. Enter a value for *Schedule*. In this example, the value is `*/2 * * * *`, which creates a PingSource that sends a message every two minutes.
+.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*. The  *Event Sources* page in the *Developer Catalog* is displayed.
```

Comment:
The  **Event Sources** page is displayed.

Reply:
Done.